### DISCUSSION TITLE
Define LAG Members during Interface Bulk Import

### DISCUSSION THREAD

**Discussion by Daniel-Dietz** [netbox-community/netbox#9658] [Q&A]
Hi,

is it possible to define the LAG Members during an import? 

Thanks,
Daniel

**Comment by candlerb** [comment]
It depends exactly what you're trying to import.

When importing interfaces, in principle it is possible.  At `/dcim/interfaces/import/` you'll see one of the available columns is `lag` - the name of the lag to which this interface is attached.

Therefore, you should be able to import interface "lag1" followed by "Gi0/1" followed by "Gi0/2", where "Gi0/1" and "Gi0/2" are members of "lag1".

Unfortunately, CSV import cannot be used to _modify_ any object - only create a new one.  So if the interfaces Gi0/1 and Gi0/2 already exist, you're out of luck unless you delete them first and then import them from scratch (which will also delete any other attributes of those interfaces, not to mention any cables which are connected to them)

Therefore: if you want to import just an interface "lag1", and then set Gi0/1 and Gi0/2 to point to lag1, then you'll need to use the REST API or a custom script to do the latter.
